packages/core/test/test-util/filter-factory.ts

```diff
@@ -1,6 +1,6 @@
 import { v4 as uuid } from 'uuid';
 import { or } from '../../src';
-import { all, any, filterFunctionsV4 } from '../../src/odata-v4';
+import { all, any, filterFunctions } from '../../src/odata-v4';
```

Comment:
Maybe you could do `filterFunctions as filterFunctionsV4`, just because in other places we do that too. This will make it easier to spot this later on, when we split the repos.

Reply:
Good catch, updated it